src/components/maincomponents/Header.js

```diff
@@ -19,6 +22,9 @@ export function Header() {
               Favorite
             </NavLink>
           </NavItem>
+          <NavItem style={{ color: "yellow" }}>
```

Comment:
The number is not aligned vertically with the links:

<img width="115" alt="image" src="https://user-images.githubusercontent.com/6420535/168234459-a1a8f142-830d-4f0f-a48a-62beab416617.png">

To place non-link elements in the nav bar, you can use the `<NavbarText>` component from Reactstrap and it will have the correct padding.

Reply:
Thank you, i will change it